php/utils.php

```diff
@@ -823,3 +823,16 @@ function parse_str_to_argv( $arguments ) {
 	}, $argv );
 	return $argv;
 }
+
+/**
+ * Locale-independent version of basename()
+ *
+ * @access public
```

Comment:
`@access public` is how some utils end up here: http://wp-cli.org/docs/internal-api/